lib/trento/databases/database.ex

```diff
@@ -499,6 +515,21 @@ defmodule Trento.Databases.Database do
 
   defp maybe_emit_database_instance_registered_event(_, _), do: nil
 
+  defp maybe_emit_database_tenants_updated_event(
+         %Database{database_id: database_id, tenants: database_tenants},
```

Comment:
I would rename this to `current_tenants` and the other to `new_tenants`.
It looks more precise.
We don't have anything like instance tenants in reality

Reply:
done!